badge-maker/lib/color.spec.js

```diff
@@ -37,7 +37,7 @@ test(normalizeColor, () => {
   given('  blue ').expect('  blue ')
   given('rgb(100%, 200%, 222%)').expect('rgb(100%, 200%, 222%)')
   given('rgb(122, 200, 222)').expect('rgb(122, 200, 222)')
-  given('rgb(100%, 200, 222)').expect('rgb(100%, 200, 222)')
```

Comment:
This was the non compliant example that I flagged up in the last bullet point of https://github.com/badges/shields/pull/5508#issuecomment-690930346.